images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -11,38 +11,77 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     PYTHONIOENCODING="UTF-8" \
     LANG="C.UTF-8" \
     LANGUAGE="C.UTF-8" \
-    LC_ALL="C.UTF-8"
-
-ARG IMAGES_DIR="/opt/tmp/images"
-RUN oryx prep --skip-detection --platforms-and-versions nodejs=14 --debug \
-    && echo "$DEBIAN_FLAVOR" \
-    && . /tmp/build/__goVersions.sh \
-    && downloadedFileName="go${GO_VERSION}.linux-amd64.tar.gz" \
-    && ${IMAGES_DIR}/retry.sh "curl -SLsO https://golang.org/dl/$downloadedFileName" \
-    && mkdir -p /usr/local \
-    && gzip -d $downloadedFileName \
-    && tar -xf "go${GO_VERSION}.linux-amd64.tar" -C /usr/local \
-    && rm -rf $downloadedFileName
-
-RUN set -ex \
-    # Install Python SDKs
-    # Upgrade system python
-    && PYTHONIOENCODING="UTF-8" \
-    # It's not clear whether these are needed at runtime...
-    && apt-get update \
+    LC_ALL="C.UTF-8" \
+    ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin"
+    
+RUN apt-get update \
```

Comment:
Could you please clarify if the `RUN set -ex \` was removed on purpose? It seems still valuable as we add it in most scripts to exit on first failure and print the commands used, but maybe in this case we don't want that anymore?

Reply:
Ideally we should be keeping `set -ex` since it is helpful for debugging and avoiding false negatives.

Reply:
Updated Dockerfile